core/src/main/java/org/hisp/dhis/android/core/dataset/internal/DataSetInstanceSQLStatementBuilder.kt

```diff
@@ -207,7 +205,6 @@ open class DataSetInstanceSQLStatementBuilder : ReadOnlySQLStatementBuilder {
             joinDataSetCompleteRegistration
 
         private val INNER_SELECT_CLAUSE = "SELECT " +
-            DATAVALUE_ID + AS + DeletableDataColumns.ID + "," +
```

Comment:
This is only used to assign a `_id` value to DataSetInstance objects. This model extends from CoreObject, but it doesn't map to a table. I have verified that the only consequence of this change is that the returned values have a null `_id` property.